@here/harp-mapview/lib/MapView.ts

```diff
@@ -2001,6 +2006,35 @@ export class MapView extends THREE.EventDispatcher {
         this.update();
     }
 
+    /**
+     * Updates the value of a dynamic property.
+     *
+     * the style operator `["get", "property name", ["dynamic-properties"]]`.
+     *
+     * @param name The name of the property.
```

Comment:
Please add a comment that name must not start with `$`

Reply:
done